src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,16 @@
 
 public class MainApp {
 
+    public static void main(String[] args) {
+        Machine[] machines = new Machine[] {
+                new Bulldozer("Buldozer"),
+                new Excavator("Excavator"),
+                new Truck("Truck")
```

Comment:
It seems like you are passing a string parameter to the constructors of `Bulldozer`, `Excavator`, and `Truck`, which is not necessary for the task. Instead, the message should be generated within the `doWork` and `stopWork` methods of each machine type.